readme.md

```diff
@@ -26,7 +38,7 @@ Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
 ```text
-
+Just like we use jquery to make ajax get request to 3rd party api to get json data in the above example, jquery is used in similar fashion to submit a form using the "POST" datatype. the format is similar - so using jquery, ajax can perform all crud requests.
```

Comment:
On the right track, but looking for a bit more info, such as using jQuery to get the values from the form, package it up into an object, and sending that object with the POST request.